webpack/assets/javascripts/react_app/components/common/TypeAheadSelect/TypeAheadSelectSelectors.js

```diff
@@ -0,0 +1,18 @@
+import Immutable from 'seamless-immutable';
+
+export const mapSelected = selected => selected.map(item => item.label || item);
+
+const selectTypeAheadSelect = ({ typeAheadSelect }, id) =>
+  typeAheadSelect[id] || {};
+
+export const typeAheadSelectExists = ({ typeAheadSelect }, id) =>
```

Comment:
```suggestion
export const selectTypeAheadSelectExists = ({ typeAheadSelect }, id) =>
```
Prefer selector naming consistent with all other selectors.  I know you hate it but I just made @xprazak2 do the same thing so I feel like I have to be consistent :yum:

Reply:
Np. I think you suggested that name originally though ;) (Sorry if wrong)